src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('ul li');
```

Comment:
Ensure that the 'ul li' elements are present in the DOM before querying them. If this script runs before the DOM is fully loaded, it might not find any elements.